Source/SVWidgetsLib/Widgets/util/AddFilterCommand.cpp

```diff
@@ -301,9 +302,13 @@ void AddFilterCommand::disconnectFilterSignalsSlots(const AbstractFilter::Pointe
   {
     return;
   }
- // PipelineModel* model = m_PipelineView->getPipelineModel();
 
   QObject::disconnect(filter.get(), &AbstractFilter::filterCompleted, nullptr, nullptr);
 
   QObject::disconnect(filter.get(), &AbstractFilter::filterInProgress, nullptr, nullptr);
+
+  for(auto& connection : m_Connections)
+  {
+    QObject::disconnect(connection);
+  }
```

Comment:
This method is called per filter so having a map to the filter might make more sense here, but m_Connections needs to be cleared after disconnecting the connection.  Otherwise, m_Connections will continue to grow larger every time undo() redo() are called.

Reply:
This is duplicated in RemoveFilterCommand.